src/dynafed.rs

```diff
@@ -538,9 +593,11 @@ mod tests {
             fedpegscript: vec![0x06, 0x07],
             extension_space: vec![vec![0x08, 0x09], vec![0x0a]],
         };
+        println!("full debug: {:?}", full);
```

Comment:
Can we do String compare instead of printing? We do not have prints in any of the other tests and it would be helpful for print debugging if we don't have print statements.

Reply:
donw